training/distributed_training/pytorch/data_parallel/maskrcnn/pytorch_smdataparallel_maskrcnn_demo.ipynb

```diff
@@ -8,7 +8,7 @@
     "\n",
```

Comment:
<p>I can see this section will become a subject of confusion for customers with less experience in network engineering. In particular, things like "appropriate inbound/outbound" rules should be replaced by the precise rules needed for this notebook and even better, why those rules are needed. </p><p> </p><p>One suggestion is you use boto3 calls to programmatically configure the security group, so that people can read your code and figure out what rules needs to be attached to the security group and what ports need to be open. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/2087/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>